### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by igorcarrasco** [madMAx43v3r/chia-plotter#841]
I'm using ubuntu's last stable version
and I followed the steps from this video: https://www.youtube.com/watch?v=h-TXA9eGn6k&t=0s
so I'm using this branch:
git clone -b pool-puzzles https://github.com/madMAx43v3r/chia-plotter

./build/chia_plot -n -1 -c ********************************* -f ********************************-t /mnt/sas2300/ -d /media/utiger/E12/ -r 8 -u 256 -w

Both the temp and destination dir have been used with no issues for normal chia plotting with plotman for weeks.

This problem Segmentation fault (core dumped) has happened 3 times, I have been unable to finish a single plot.
Any tips?
Build:
ryzem 3700x
64 GB DDR4 3200

**Comment by ataa** [comment]
Unstable OC, Low DIMM Voltage, Bad Memory Module or even Kernel OOM (WSL) can cause this. Search in the issues.

**Comment by igorcarrasco** [comment]
I've been using it for months without issues.
Is madmax more demanding for ram or something?

**Comment by ataa** [comment]
Exactly, Start with Memtest. You can find it in Grub menu in Ubuntu.
